SimCalorimetry/HGCalSimProducers/src/HGCFEElectronics.cc

```diff
@@ -54,6 +54,12 @@ HGCFEElectronics<DFr>::HGCFEElectronics(const edm::ParameterSet &ps) :
     }
   if( ps.exists("adcThreshold_fC") )                adcThreshold_fC_                = ps.getParameter<double>("adcThreshold_fC");
   if( ps.exists("tdcOnset_fC") )                    tdcOnset_fC_                    = ps.getParameter<double>("tdcOnset_fC");
+  if( ps.exists("tdcForToaOnset_fC") ){
+    auto temp = ps.getParameter< std::vector<double> >("tdcForToaOnset_fC");
+    for( unsigned i = 0; i < temp.size(); ++i ) {
```

Comment:
use `std::copy()` or `std::copy_n()` to do this
also do some bounds checking
(same for the next two below)